src/daft-scan/src/lib.rs

```diff
@@ -119,7 +125,7 @@ impl ChunkSpec {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Hash)]
```

Comment:
we should skip hashing `parquet_metadata` since it's considered more of a "cached" field to allow us to skip fetching it again. This should allow you to avoid changes in parquet2 crate.

Reply:
Sounds good, removed all changes to parquet2